### PR TITLE
(chore) Add release script

### PR DESCRIPTION
**Pull request by denniskigen**
Adds a release script that can be used to cut releases of this project.

### CODE REVIEW

**Comment by denniskigen** [comment]
@ibacher, what do you reckon would be a good version number to start releasing this project at?

**Comment by ibacher** [comment]
Honestly, I think 1.0.0 is fine... Releasing things starting at 3.0.0 was a mistake...

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "start": "openmrs develop",
     "serve": "webpack serve --mode=development",
     "build": "webpack --mode production",
+    "release": "lerna version --no-git-tag-version",
```
Comment (ibacher) [inline package.json:13]: This isn't a lerna repo is it? Is this going to work?

Comment (denniskigen) [reply package.json:13]: Oops, no.

**Pull request closed without merging**